Fatras/include/ActsFatras/EventData/Hit.hpp

```diff
@@ -88,9 +88,9 @@ class Hit {
   }
   /// Average normalized particle direction vector through the surface.
   Vector3 unitDirection() const {
```

Comment:
just for reference - it might be better to not average the direction but stick to either before or after. I think for digitization we should use before?